src/d3a/models/strategy/storage.py

```diff
@@ -192,18 +194,20 @@ def _area_reconfigure_prices(self, **kwargs):
                       f"Traceback: {traceback.format_exc()}")
             return
 
-        self.offer_update.initial_rate_profile_buffer = initial_selling_rate
-        self.offer_update.final_rate_profile_buffer = final_selling_rate
-        self.bid_update.initial_rate_profile_buffer = initial_buying_rate
-        self.bid_update.final_rate_profile_buffer = final_buying_rate
-        self.bid_update.energy_rate_change_per_update_profile_buffer = \
-            energy_rate_increase_per_update
-        self.offer_update.energy_rate_change_per_update_profile_buffer = \
-            energy_rate_decrease_per_update
-        self.bid_update.fit_to_limit = bid_fit_to_limit
-        self.offer_update.fit_to_limit = offer_fit_to_limit
-        self.bid_update.update_interval = update_interval
-        self.offer_update.update_interval = update_interval
+        self.offer_update.update_members(
```

Comment:
Minor, but I think that a name like `set_parameters()` would be much more appropriate.